### PR TITLE
Fix the check for processed envelope

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Fix the check for processed envelope.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/StatusTest.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
     @Test
     public void notification_sent_or_processed_only_is_processed() {
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/StatusTest.java]: method name needs update ?

Comment (lgonczar) [reply src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/StatusTest.java]: true, done

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
     }
 
     public boolean isProcessed() {
-        return this == PROCESSED || this == NOTIFICATION_SENT;
+        return this == PROCESSED || this == NOTIFICATION_SENT || this == COMPLETED;
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java]: can you explain?
what does 'processed' actually mean and when is it used?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java]: adding javadoc to this method would be great, I don't think it's clear with those 3 different statuses, out of which one is called 'processed'

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java]: also, probably good idea to rename this method

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java]: I'll add a comment. The name of me method may be right - all those statuses represent a processed envelope, from processor's perspective. And `PROCESSED` itself is probably going away, as it duplicates another status - `UPLOADED`.

**Pull request merged**